src/main/java/org/jenkinsci/plugins/changeassemblyversion/ChangeAssemblyVersion.java

```diff
@@ -118,16 +182,23 @@ public String getAssemblyCulture() {
      *
      */
     @Override
-    public boolean perform(AbstractBuild build, Launcher launcher, BuildListener listener) {
+    public void perform(Run<?, ?> run, FilePath workspace, Launcher launcher, TaskListener listener)
```

Comment:
Probably ought to have a method matching the original signature that delegates to the new perform, to preserve binary compatibility.   Unless you can prove there are no plugins depending on this one.

Reply:
New (old) method signature added.